### PR TITLE
Restore FetchAs with beta Thoth.Json

### PR DESCRIPTION
**Pull request by rfrerebe**
#1

### CODE REVIEW

File: src/Fable.Fetch.fsproj [hunk 1]
```diff
@@ -16,5 +16,6 @@
     <PackageReference Include="Fable.Core" Version="2.0.2" />
     <PackageReference Include="Fable.Import.Browser" Version="1.3.0" />
     <PackageReference Include="Fable.Promise" Version="1.0.0" />
+    <PackageReference Include="Thoth.Json" Version="3.0.0-beta-002" />
```
Comment (MangelMaxime) [inline src/Fable.Fetch.fsproj:19]: We should not release a stable version using a beta dependency

**Review by MangelMaxime — CHANGES_REQUESTED** [review]
We removed Thoth.Json depency from Fable.Fetch so people could chose the JSON library they want to use.

Like Fable.SimpleJson, Thoth.Json, their own implementation.

So I am not sure, we want to re-add it to the library or if we want a new package.

File: src/Fable.Fetch.fsproj (hunk 1 above)
Comment (rfrerebe) [reply src/Fable.Fetch.fsproj:19]: Sure, my idea was to also release a beta version if merged 😄

**Comment by rfrerebe** [comment]
Ok, I have no issue dropping this PR, but I would welcome a shared vision from fable, safe-stack and Thoth (you), because some packages are deprecated [Fable.PowerPack](https://github.com/fable-compiler/Fable-powerpack), but still referenced in [safe-stack documentation](https://safe-stack.github.io/docs/feature-clientserver-serialization/) and it's not clear from the outside what is the upgrade path.

I did not know about Fable.SimpleJson, and I don't seems to be the only one to find it confusing : https://github.com/Zaid-Ajaj/Fable.SimpleJson/issues/16

I also just learnt about https://github.com/fable-compiler/fable-compiler.github.io/issues/32
which seems a nice solution.

**Comment by alfonsogarciacaro** [comment]
Actually I wanted to promote more [Fable.SimpleHttp](https://github.com/Zaid-Ajaj/Fable.SimpleHttp) instead of this package, as the API is more idiomatic in F# (specially when combining headers, etc), doesn't require the Fetch polyfill for old browsers and works with F# async.

But I know so many changes in the recommendarions are confusing for users starting the SAFE stack, so I'm not sure 🤔 

Anyway, we can try to finish first https://github.com/fable-compiler/fable-compiler.github.io/issues/32 and see if this helps users (and ourselves!) have a better understanding of the Fable ecosystem 😸

**Comment by rfrerebe** [comment]
Thanks for your answers.

I am looking forward to this new **packages.fable.io** website.
The fact that Fable.SimpleHttp is more idiomatic is not really a criteria from my point of view, I am more interested in bundle size and if it gets the job done 😄 

I am closing this PR.

**Pull request closed without merging**